scripts/experimental_deploy.py

```diff
@@ -132,7 +132,7 @@ def preprocess_release():
     subprocess.check_output(['python', 'scripts/build.py'])
 
     # Deploy to GAE.
-    subprocess.check_output([APPCFG_PATH, 'update', '.', '--oauth2'])
+    subprocess.check_output([APPCFG_PATH, 'update', '.'])
```

Comment:
Yep! That requirement was deprecated between 1.9.19 and 1.9.50. (There was a note in the terminal logs explaining this.)